src/App.js

```diff
@@ -10,10 +12,29 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    selectedTab: tabs[0],
+  }
+
+  selectTab = (selectedTab) => {
+    this.setState({
+      selectedTab: tabs.find(tab => tab.id === selectedTab.id),
+    });
+  }
```

Comment:
```suggestion
  selectTab = (selectedTab) => {
    this.setState({
      selectedTab,
    });
  }
```
ты же принимаешь в параметре selectedTab. ее и устанавливай как selected. зачем заново искать ее через find